arangod/Views/LoggerView.cpp

```diff
@@ -35,6 +35,8 @@
 
 using namespace arangodb;
```

Comment:
No longer necessary. Please remove.

Reply:
unfortunately there is still quite a lot of code in the file that does not specify a namespace and without the aforementioned the file does not colmpile

Reply:
Okay, sure, looks like you need `using arangodb::LogLevel;` inside `LevelStringToEnum` and `LevelEnumToString` and to prefix `Logger::VIEWS` with `arangodb::` at line 81. Either way, you've enclosed the rest of the implementation in the namespace, so the top-level `using namespace` isn't necessary, and could be confusing.